packages/iTwinUI-react/src/core/Typography/Text/Text.tsx

```diff
@@ -37,9 +37,8 @@ type TextOwnProps = {
   isSkeleton?: boolean;
 };
 
-export type TextProps<
-  T extends React.ElementType = 'div'
-> = PolymorphicComponentProps<T, TextOwnProps>;
+export type TextProps<T extends React.ElementType = 'div'> =
```

Comment:
Huh?

Reply:
A lot of the recent PRs have been having formatting changes such as this one